### DISCUSSION TITLE
Has the default Termux font changed since last version? (Ugly)

### DISCUSSION THREAD

**Discussion by nathaneltitane** [termux/termux-app#2563] [Q&A] [answered]
Was wondering: has the preloaded font changed since 116/117?
The font really does not render well on this last release and may benefit being either rolled back to whatever it was or simply changed to something a tad more pleasing and less harsh on the eyes?

I understand that it can easily be changed by the user as a personal preference as well, but this is just a plain sore to the eyes as it is right now.

![152278291-f6ef9d48-d2b8-4171-b23a-c349f9b2c6c6](https://user-images.githubusercontent.com/5674757/152367586-b7d653d0-d31f-4946-b400-55fd0d36e623.jpg)

**Comment by agnostic-apollo** [comment, marked as answer]
Font has not changed from android's default monospace font. Issue is likely due to samsung Android 12 One UI 4 update. Check #2539.

**Reply by nathaneltitane** [reply]
Thanks
